homeassistant/components/amcrest/binary_sensor.py

```diff
@@ -48,111 +47,93 @@ class AmcrestSensorEntityDescription(BinarySensorEntityDescription):
 SCAN_INTERVAL = timedelta(seconds=BINARY_SENSOR_SCAN_INTERVAL_SECS)
 _ONLINE_SCAN_INTERVAL = timedelta(seconds=60 - BINARY_SENSOR_SCAN_INTERVAL_SECS)
 
-_AUDIO_DETECTED_KEY = "audio_detected"
-_AUDIO_DETECTED_POLLED_KEY = "audio_detected_polled"
-_AUDIO_DETECTED_NAME = "Audio Detected"
-_AUDIO_DETECTED_EVENT_CODES = {"AudioMutation", "AudioIntensity"}
+AUDIO_DETECTED_KEY = "audio_detected"
+AUDIO_DETECTED_POLLED_KEY = "audio_detected_polled"
+AUDIO_DETECTED_NAME = "Audio Detected"
+AUDIO_DETECTED_EVENT_CODES = {"AudioMutation", "AudioIntensity"}
 
-_CROSSLINE_DETECTED_KEY = "crossline_detected"
-_CROSSLINE_DETECTED_POLLED_KEY = "crossline_detected_polled"
-_CROSSLINE_DETECTED_NAME = "CrossLine Detected"
-_CROSSLINE_DETECTED_EVENT_CODE = "CrossLineDetection"
+CROSSLINE_DETECTED_KEY = "crossline_detected"
+CROSSLINE_DETECTED_POLLED_KEY = "crossline_detected_polled"
+CROSSLINE_DETECTED_NAME = "CrossLine Detected"
+CROSSLINE_DETECTED_EVENT_CODE = "CrossLineDetection"
 
-_MOTION_DETECTED_KEY = "motion_detected"
-_MOTION_DETECTED_POLLED_KEY = "motion_detected_polled"
-_MOTION_DETECTED_NAME = "Motion Detected"
-_MOTION_DETECTED_EVENT_CODE = "VideoMotion"
+MOTION_DETECTED_KEY = "motion_detected"
+MOTION_DETECTED_POLLED_KEY = "motion_detected_polled"
+MOTION_DETECTED_NAME = "Motion Detected"
+MOTION_DETECTED_EVENT_CODE = "VideoMotion"
 
-_ONLINE_KEY = "online"
+ONLINE_KEY = "online"
 
 BINARY_SENSORS: tuple[AmcrestSensorEntityDescription, ...] = (
     AmcrestSensorEntityDescription(
-        key=_AUDIO_DETECTED_KEY,
-        name=_AUDIO_DETECTED_NAME,
+        key=AUDIO_DETECTED_KEY,
+        name=AUDIO_DETECTED_NAME,
         device_class=BinarySensorDeviceClass.SOUND,
-        event_codes=_AUDIO_DETECTED_EVENT_CODES,
+        event_codes=AUDIO_DETECTED_EVENT_CODES,
     ),
     AmcrestSensorEntityDescription(
-        key=_AUDIO_DETECTED_POLLED_KEY,
-        name=_AUDIO_DETECTED_NAME,
+        key=AUDIO_DETECTED_POLLED_KEY,
+        name=AUDIO_DETECTED_NAME + " Polled",
         device_class=BinarySensorDeviceClass.SOUND,
-        event_codes=_AUDIO_DETECTED_EVENT_CODES,
+        event_codes=AUDIO_DETECTED_EVENT_CODES,
         should_poll=True,
```

Comment:
Why are there polling and non-polling sensors?

Not for this PR, but can the polling ones be disabled by default?